package.json

```diff
@@ -147,3 +148,4 @@
     "printWidth": 100
   }
 }
+
```

Comment:
So it should be there, not sure what is happening but basically this file needs only `pako` change 😅

Reply:
I am a little confused by your previous comment then. I checked the current file : https://github.com/aws/amazon-chime-sdk-js/blob/release-2.x/package.json

I did not see a new line there.

So should we add a new line or I should drop it?